src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -184,6 +198,7 @@ private static JsonObject createItemContext(Item item) {
     if (location != null) {
       itemContext
         .put("effectiveLocationSpecific", location.getName())
+        .put("effectiveLocationPrimaryServicePointName", location.getPrimaryServicePoint().getName())
```

Comment:
If this changes the API (and is looks like it does - we're adding another field to the `/circulation/pick-slips/{id}` endpoint) we should increase minor version of the corresponding interface in the ModuleDescriptor. Check the 'Version numbers' section here https://dev.folio.org/guidelines/contributing/ specifically this part: _If you only add things to the interface – e.g. a new resource or method on existing resources – then you increment the minor number, because the API is backwards compatible._